### ISSUE TITLE
Greek language

### ISSUE THREAD

**Issue by manolischania** [accumulator/plugin.video.lbry#25]
it would be nice to include the Greek language so that videos with Greek titles can be played

**Comment by accumulator** [comment]
I don't think there's anything blocking greek titled videos, as all text is handled as unicode. What exactly is the problem you are experiencing?

**Comment by manolischania** [comment]
> I don't think there's anything blocking greek titled videos, as all text is handled as unicode. What exactly is the problem you are experiencing?

the title is viewable by the plugin but if it has greek characters It doesn't play it also if any channel is in greek characters It's not even accessible by the plugin

**Comment by accumulator** [comment]
Ok, will have a look

**Comment by karlo2105** [comment]
I would like to point out i have same message  Failed: HTTP returned code 401 on several videos on Kodi LBRY extension which work normally on Odysee. It started on December on some videos and has expanded on all videos from several channels which worked normally previously. All those videos work flawlessy on Odysee website. 
Is there any chance to sync Kodi LBRY with Odysee subscription ?

**Comment by accumulator** [comment]
First suspecting an issue with URL encoding the URL passed to the player, I added code to re-encode the stream URL path.

However, I run into the issue @karlo2105 describes, with videos not playing, and after visiting the same video on  odysee.com using the browser, it plays in the browser, but *also* in kodi (exact same URL).

So it looks like the odysee.com website does something to make the stream available which isn't done by the API server.

For now I can only recommend running your own lbrynet server (which doesn't seem to have a problem serving the stream, although I'm running a maybe bit outdated version..)

**Comment by karlo2105** [comment]
I found a workaround by installing LBRY app and let it in background. In LBRY addon, i replaced public server by http://localhost:5279  It loads faster and all videos play. 
In LBRY app, i logged in and selected my account in LBRY addon. It was successful so i expected to get synchronised subscription list on LBRY addon but it is empty. How come doesn't it sync in LBRY addon despite success binding with LBRY app ?
Is there any workaround ?

**Comment by accumulator** [comment]
> I found a workaround by installing LBRY app and let it in background. In LBRY addon, i replaced public server by http://localhost:5279 It loads faster and all videos play. 

Yes, like I described above, if you run your own lbrynet server it can resolve the stream url.

> In LBRY app, i logged in and selected my account in LBRY addon. It was successful so i expected to get synchronised subscription list on LBRY addon but it is empty. How come doesn't it sync in LBRY addon despite success binding with LBRY app ? Is there any workaround ?

Subscription list is currently local only. (TODO)
